tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_case_study.html

```diff
@@ -0,0 +1,53 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% image featured_case_study_image fill-828x546 format-webp as featured_desktop_image %}
```

Comment:
We need to fall back to a feed image if there isn't a case study image defined - but maybe that check could be done in the back-end and pass just one image through to the template. cc @engineervix

Reply:
Cool, will leave it as is for now